vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/gridProConnector.js

```diff
@@ -8,6 +8,59 @@
     }
 
     window.Vaadin.Flow.gridProConnector = {
+        initLazy: grid => tryCatchWrapper(function(grid) {
```

Comment:
There's a lot of private API access in these changes because the feature doesn't exist in the Web Component. In order to ship a new feature in a sustainable manner, it first needs to be implemented in the Web Component behind a well-designed API, which can then be used in the Flow component implementation.